helm/secrets-provider/values.schema.json

```diff
@@ -112,6 +112,16 @@
               "type": "string",
               "minLength": 1
             },
+            "timeout": {
+              "type": "number",
+              "minLength": 1,
+              "minimum": 5
+            },
+            "retryCountLimit": {
+              "type": "number",
+              "minLength": 1,
+              "minimum": 1
```

Comment:
Why 0 is not a valid retry limit?
I may want to not retry at all.

Reply:
@oburstein-hub ^

Reply:
because it will create busy waiting